src/linker/cli.py

```diff
@@ -26,6 +26,10 @@ def linker():
     "pipeline_specification",
     type=click.Path(exists=True, dir_okay=False, resolve_path=True),
 )
+@click.argument(
+    "input_data",
+    type=click.Path(exists=True, dir_okay=False, resolve_path=True),
+)
```

Comment:
Need help text saying that this is a path to a config file, not the data itself.

Reply:
click does not support the `help` argument for click.arguments (to keep w/ unix standard). Will document in the `run` docstring